### PR TITLE
Add default replace for - to _

### PR DESCRIPTION
**Pull request by mattmurch**
Also updated changelog and docs.

Resolves: #549

### CODE REVIEW

**Comment by sampsyo** [comment]
Thanks for the PR! But is #459 really what you meant to link to?

**Comment by mattmurch** [comment]
Sorry, I meant #549. I'll push a fix in a second.

**Comment by mattmurch** [comment]
Done

**Comment by mattmurch** [comment]
There is a Travis CI failure in the line I introduced in the changelog. I wrote line 55 to mimic the surrounding code, but I do not understand what it is doing. Do you know what is happening?

File: docs/changelog.rst [hunk 1]
```diff
@@ -52,6 +52,7 @@ New features:
 
 Fixes:
 
+* :doc:`/beets/config_default`: Added default replace for - to _. :bug:`549`
```
Comment (sampsyo) [inline docs/changelog.rst]: Aha; this is just a ReST syntax thing: the `:doc:` directive is for linking to other `*.rst` pages. We actually only really use this in the changelog for plugins, where the name of the ReST doc is the same as the name of the Python source file—here, you don't need to use a prefix. You might want to use ``:ref:`replace` `` inline in the sentence to link to the relevant config option in the configuration page, for example.

**Comment by sampsyo** [comment]
Thanks! Merged, with a slightly updated changelog entry.

**Pull request merged**